docs/Common/Enums/index.md

```diff
@@ -529,6 +529,7 @@
 |USER_EXIT|1|By getting this value, SDL puts the named app to NONE HMILevel|
 |UNAUTHORIZED_TRANSPORT_REGISTRATION|2|By getting this value, SDL unregisters the named application|
 |UNSUPPORTED_HMI_RESOURCE|3|By getting this value, SDL unregisters the named application|
+|CLOSE_CLOUD_CONNECTION|4|By getting this value, SDL puts the named app to NONE HMILevel. Used by the HMI to close a cloud app connection|
```

Comment:
Update the onExitApplication RPC page to describe the outcome of sending this message. ie App will be unregistered but an updateDevicelist will also be sent with the closed application. The closed application will have a cloudConnectionStatus of NOT_CONNECTED